tests/integ/test_pandas_to_df.py

```diff
@@ -295,6 +303,88 @@ def test_create_dataframe_from_pandas(session):
     # assert_frame_equal(results, pd, check_dtype=False)
 
 
+def test_create_dataframe_from_pandas_with_schema(session):
```

Comment:
can we also try time-related types and some structured types?

Reply:
Time-related types requires enabling parameter that controls parquet timestamp behavior, otherwise I'm hitting `E               snowflake.snowpark.exceptions.SnowparkFetchDataException: (1406): Failed to fetch a Pandas Dataframe. The error is: Casting from timestamp[us] to timestamp[ns] would result in out of bounds timestamp: 5325833340940354560
`. Looks like the similar test is also disabled in `test_create_dataframe_from_pandas`. 

Should we wait until new logical type is rolled out in production?

Reply:
hmm sure, let's have a jira for that? (and when will new logical type be in production? )

Reply:
I asked Yucan but haven't heard back yet. Will create a JIRA that covers both this test and `test_create_dataframe_from_pandas`.

Reply:
Unfortunately we might not be able to test for semi-structured types as well due to connector returning json strings :/ Will create another JIRA to track this test TODO as well.